### ISSUE TITLE
Ground Vehicles switch

### ISSUE THREAD

**Issue by lloydbayley** [fratsloos/fr24_card#23]
G'day,

I was thinking that a filter on ground-based detections might be a good idea.
There are a few that I pick up that crawl around on the ground here and there (must be test vehicles/transmitters) and I'd love to get rid of them on my listing. Perhaps a "ground_vehicles: false" kind of thing perhaps?
Or perhaps work it into an altitude filter if one is going to be implemented.

Just a thought!

Cheers!

**Comment by fratsloos** [comment]
It's a good idea, but the thing is that I don't see any ground vehicles on my radar. That's because the position of my radar is not optimised; there is no airport within range in the direction that my radar can detect.

Do the ground vehicles have some sort of different identification in the data coming from the radar? If so, could you share the data? An altitude filter is possible, but then it will also filter the airplanes that are on the same altitude, eg the airplanes that are departing, landing or taxiing.

**Comment by lloydbayley** [comment]
These are the little buggers. They have N/A for the registration and ground for the altitude.
I agree, which way to go. If you did both, there COULD be a  real 'N/A' aircraft on the 'ground' but it's probably very-unlikely.
It would be a 'fix' but a bit of band-aid.

It's a tricky one!

<img width="482" alt="Screenshot 2023-11-25 at 4 57 45 pm" src="https://github.com/fratsloos/fr24_card/assets/21309308/65735624-631f-481d-867e-82841b807fe1">

**Comment by fratsloos** [comment]
Thanks for the example. Can you share the JSON data?

You can find it by opening the URL of your REST sensor in your browser. Then copy and paste it.

**Comment by lloydbayley** [comment]
It ain't pretty but it's full of useful info! :)

```
{ "now" : 1701103675.5,
  "messages" : 15055426,
  "aircraft" : [
    {"hex":"7c66e2","alt_baro":35050,"alt_geom":35900,"gs":448.7,"ias":264,"tas":450,"mach":0.780,"track":38.1,"track_rate":0.00,"roll":-0.2,"mag_heading":16.2,"baro_rate":-32,"geom_rate":0,"squawk":"1464","nav_qnh":1013.0,"nav_altitude_mcp":35008,"lat":-34.102890,"lon":148.178437,"nic":8,"rc":186,"seen_pos":12.5,"version":0,"nac_p":8,"nac_v":0,"sil":2,"sil_type":"unknown","mlat":[],"tisb":[],"messages":31,"seen":11.4,"rssi":-26.4},
    {"hex":"7c764b","flight":"QFA7305 ","alt_baro":34000,"alt_geom":35000,"gs":409.0,"ias":252,"tas":424,"mach":0.732,"track":219.8,"track_rate":0.03,"roll":0.2,"mag_heading":216.6,"baro_rate":32,"geom_rate":64,"squawk":"1022","category":"A0","nav_altitude_mcp":34000,"lat":-32.422073,"lon":148.937311,"nic":8,"rc":186,"seen_pos":0.2,"version":0,"nac_p":8,"nac_v":0,"sil":2,"sil_type":"unknown","mlat":[],"tisb":[],"messages":3071,"seen":0.2,"rssi":-20.3},
    {"hex":"c82aa5","flight":"TFX203  ","alt_baro":38850,"alt_geom":40050,"gs":455.4,"ias":242,"tas":448,"mach":0.780,"track":37.1,"track_rate":0.00,"roll":0.0,"mag_heading":15.1,"baro_rate":768,"geom_rate":736,"squawk":"1505","emergency":"none","category":"A3","nav_qnh":1013.6,"nav_altitude_mcp":39008,"nav_altitude_fms":39008,"nav_heading":16.2,"lat":-32.744518,"lon":149.428524,"nic":8,"rc":186,"seen_pos":4.1,"version":2,"nic_baro":1,"nac_p":9,"nac_v":2,"sil":3,"sil_type":"perhour","gva":2,"sda":2,"mlat":[],"tisb":[],"messages":8334,"seen":0.4,"rssi":-24.6},
    {"hex":"7c5b29","flight":"QFA7294 ","alt_baro":8425,"alt_geom":8800,"gs":297.9,"track":71.2,"baro_rate":-1408,"squawk":"4367","category":"A0","nav_qnh":1013.2,"lat":-34.273590,"lon":150.918965,"nic":8,"rc":186,"seen_pos":4.2,"version":2,"nic_baro":1,"nac_p":9,"nac_v":0,"sil":3,"sil_type":"perhour","gva":2,"sda":2,"mlat":[],"tisb":[],"messages":4818,"seen":1.9,"rssi":-23.8},
    {"hex":"7c404c","flight":"MZI     ","alt_baro":11000,"alt_geom":11500,"gs":177.3,"track":15.4,"geom_rate":0,"squawk":"1251","emergency":"none","category":"A2","lat":-33.138909,"lon":150.829898,"nic":8,"rc":186,"seen_pos":0.2,"version":2,"nic_baro":0,"nac_p":9,"nac_v":2,"sil":3,"sil_type":"perhour","gva":2,"sda":2,"mlat":[],"tisb":[],"messages":9830,"seen":0.2,"rssi":-18.7},
    {"hex":"7cf7ca","flight":"SSM1    ","alt_baro":"ground","gs":2.3,"track":206.6,"baro_rate":0,"squawk":"7712","lat":-33.870710,"lon":151.209126,"nic":0,"rc":0,"seen_pos":7.5,"nac_p":0,"nac_v":0,"sil":0,"sil_type":"unknown","mlat":["gs","track","baro_rate","lat","lon","nic","rc","nac_p","nac_v","sil","sil_type"],"tisb":[],"messages":119023,"seen":0.4,"rssi":-22.7},
    {"hex":"7cf7cc","flight":"SSM3    ","alt_baro":"ground","gs":11.1,"track":95.2,"baro_rate":0,"squawk":"7727","lat":-33.547405,"lon":150.618896,"nic":0,"rc":0,"seen_pos":4.6,"nac_p":0,"nac_v":0,"sil":0,"sil_type":"unknown","mlat":["gs","track","baro_rate","lat","lon","nic","rc","nac_p","nac_v","sil","sil_type"],"tisb":[],"messages":350320,"seen":0.2,"rssi":-2.8}
  ]
}
```

**Comment by fratsloos** [comment]
Today I released version 1.0.0-alpha. This release includes changes for this request. If you want to test the feature, please see the [release](https://github.com/fratsloos/fr24_card/releases/tag/v1.0.0-alpha). To use this feature, the card config needs to be updated, see the [readme](https://github.com/fratsloos/fr24_card/blob/v1.0.0-alpha/readme.md) for more information.

The card can be tested by enabling the option 'Show beta versions' when (re)downloading the card in HACS. Issues with the release can be reported in the issue tracker.

**Comment by lloydbayley** [comment]
Thanks @fratsloos ! Working for me without the pesky ground vehicles! 
Will keep an eye on it and watch out for any possible bugs. So far, so good!
